From ea407fb3143ae4b56e6a53f0a6b37c85bb09bffe Mon Sep 17 00:00:00 2001
From: "commit-queue@webkit.org"
 <commit-queue@webkit.org@268f45cc-cd09-0410-ab3c-d52691b4dbfc>
Date: Tue, 14 Mar 2017 18:53:32 +0000
Subject: [PATCH] Remove redundant check for "firstLine" in
 RenderBlock::lineHeight() https://bugs.webkit.org/show_bug.cgi?id=169610

Patch by Adrian Perez de Castro <aperez@igalia.com> on 2017-03-14
Reviewed by Michael Catanzaro.

* rendering/RenderBlock.cpp:
(WebCore::RenderBlock::lineHeight): Remove test of "firstLine" that
was already checked in the condition for the enclosing if-clause.

git-svn-id: http://svn.webkit.org/repository/webkit/trunk@213923 268f45cc-cd09-0410-ab3c-d52691b4dbfc

diff --git a/Source/WebCore/rendering/RenderBlock.cpp b/Source/WebCore/rendering/RenderBlock.cpp
index f781c38..f71dcc8 100644
--- a/Source/WebCore/rendering/RenderBlock.cpp
+++ b/Source/WebCore/rendering/RenderBlock.cpp
@@ -2943,7 +2943,7 @@ LayoutUnit RenderBlock::lineHeight(bool firstLine, LineDirectionMode direction,
         return RenderBox::lineHeight(firstLine, direction, linePositionMode);
 
     if (firstLine && view().usesFirstLineRules()) {
-        auto& s = firstLine ? firstLineStyle() : style();
+        auto& s = firstLineStyle();
         if (&s != &style())
             return s.computedLineHeight();
     }
-- 
2.9.3

